.pre-commit-config.yaml

```
@@ -16,3 +16,17 @@ repos:
     rev: 6.0.0
     hooks:
       - id: pydocstyle
+
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.961
+    hooks:
+      - id: mypy
+        args:
+          [
+            --pretty,
+            --show-error-codes,
+            --no-strict-optional,
+            --ignore-missing-imports,
+            --install-types,
+            --non-interactive,
+          ]
```

I'm going to sneak this into this PR as well (another item I've been meaning to add):
```suggestion
          ]
    - repo: https://github.com/pycqa/isort
      rev: 5.10.1 # This should be kept in sync with the version in requirements-dev.in
      hooks:
        - id: isort
          args: ["--profile", "black", "--filter-files"]
          files: rmc
```